java/unit.go

```diff
@@ -26,18 +29,18 @@ func (self *UnitDeclaration) Import(package_ string) *UnitDeclaration {
 	return self
 }
 
-func (self *UnitDeclaration) AddDeclarations(declarations ...Writable) *UnitDeclaration {
+func (self *UnitDeclaration) AddDeclarations(declarations ...util.Writable) *UnitDeclaration {
 	self.declarations = append(self.declarations, declarations...)
 	return self
 }
 
 func (self *UnitDeclaration) Code() string {
-	writer := CreateWriter()
+	writer := util.CreateWriter(2)
```

Comment:
thats the magic number here

Reply:
Nevermind i learned to keep reading

Reply:
?

Reply:
I feel like this number be set as a constant per language, or somehow be configurable by the project importing this library. Totally doesn't need to happen here tho